src/ontology/edits/oeo-physical.omn

```diff
@@ -2874,7 +2878,7 @@ Class: OEO_00000300
         <http://purl.obolibrary.org/obo/IAO_0000119> "Derived from https://en.wikipedia.org/w/index.php?title=Nuclear_binding_energy&oldid=1007327561"@en,
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/211
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/225
-convertion to nuclear binding energy
```

Comment:
Not sure about wether this should read "conversion to nuclear binding energy" instead of "convert to nuclear binding energy". Please decide, would not block approval.